helm/tigris-stack/Chart.yaml

```diff
@@ -29,3 +29,9 @@ dependencies:
     version: 6.40.3
     repository: https://grafana.github.io/helm-charts
     condition: grafana.enabled
+  - name: metrics-server
+    version: 3.8.2
+    repository: https://kubernetes-sigs.github.io/metrics-server
+  - name: ycsb
```

Comment:
Should this be conditional for "simple" deployments such as end user trying out Tigris or deploying for Self Hosted scenarios?

Reply:
I implemented this.